src/js/airtable-list-builder.js

```diff
@@ -0,0 +1,286 @@
+(function ($, Drupal, drupalSettings) {
+  Drupal.behaviors.airtableListBuilder = {
+    attach: function (context, settings) {
+
+      // Get the content area
+      var $contentArea = $("#airtable-list");
+
+      // Grab the airtable variables.
+      let config = $contentArea.data('config');
+
+      var configDefault = {
+        "gutter": 10,
+        "equalHeight": false
+      }
+      config = {
+        ...configDefault,
+        ...config
+      }
+
+      for (filterKey in config.filters) {
+        config.filters[filterKey].choices = [{"key": "*", "name": "All"}];
+      }
+
+      // Load the airtable data
+      $.ajax({
+        type: "GET",
+        beforeSend: function (xhr) {
+          xhr.setRequestHeader('Authorization', 'Bearer iQ7GGRoNRcGi8iZf28xSgDgaDDUslteL');
+        },
+        dataType: "json",
+        url: "https://snaplogic-dev.stanford.edu/api/1/rest/feed-master/queue/StanfordDev/GSB/Research%20Hub/airtable-cache-ultra?airtable_table=" + config.table + "&airtable_view=" + config.view,
```

Comment:
I'd like to hear more about how you got this set up.

Reply:
I'll be happy to show you sometime.